### PR TITLE
[48798] sets mpi as primary source for person_types attribute

### PR DESCRIPTION
**Pull request by bramleyjl**
## Description of change

This PR is a small piece of a larger effort to move User attribute sourcing from the SSOe/User Identity to the MPI Profile; this change makes the User's MPI profile record the primary source for that user's `person_types`, with Identity remaining as a backup in case no MPI `person_type` information is found.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#48798

## Things to know about this PR

To test, authenticate with a user & artificially set their UserIdentity and MPI Profile to have different person_types, then request the user's `person_types` through the Rails console. The response should match the MPI profile unless you set it to be `nil` or an empty array, in which case the UserIdentity `person_types` value will be returned.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -243,7 +243,7 @@ def normalized_suffix
   end
 
   def person_types
-    identity_person_types.presence || mpi_person_types
```
Comment (bosawt) [inline app/models/user.rb:246]: Could we just remove the `identity_person_types` completely? Or was that one of the issues that caused a million specs to fail?

Comment (bramleyjl) [reply app/models/user.rb:246]: Sure, also returning it from `mpi_person_types` to `person_types` since it's the only source now.

**Review by bosawt — APPROVED** [review]
Mocked a user's MPI record with person types and can confirm that I see it returned with `user.person_types`

**Review by holdenhinkle — APPROVED** [review]
Read John's and Trevor's comments. 

LgTm.

**Pull request merged**